what-is-chalk.mdx

```diff
@@ -18,215 +18,481 @@ import { PyDiffEditor } from '@/components/Editor'
 
 <DocsHero />
 
-Chalk is a feature store that enables innovative machine learning teams
-to focus on building the unique products and models that make their
-business stand out. Behind the scenes, Chalk seamlessly handles
-data infrastructure with a best-in-class developer experience.
-Here’s how it works.
+Build, deploy, and iterate faster with Chalk — a programmable feature engine that powers low-latency inference, rapid model iteration, and observability across your model lifecycle.
+Chalk solves core pain points teams face when building enterprise AI and ML systems:
 
----
+- Deploying and scaling production-grade infrastructure
+- Authoring feature pipelines in pure Python — No DSLs! No rewrites!
```

Comment:
I think my same comment from before is maybe still relevant here? The header of this list is

> Chalk solves core pain points teams face when building enterprise AI and ML systems:

so when I read this bullet, I read

> Authoring feature pipelines in pure Python is a pain point teams face when building enterprise AI and ML systems

but we believe that authoring pipelines in Python is good!

wdyt?

Reply:
Fixed!